src/main/java/io/github/azagniotov/metrics/reporter/cloudwatch/CloudWatchReporter.java

```diff
@@ -756,7 +761,6 @@ public CloudWatchReporter build() {
                 metricRegistry.register("jvm.thread-states", new ThreadStatesGaugeSet());
             }
 
-            cwRateUnit = toStandardUnit(rateUnit);
```

Comment:
I would just do everything here in the `build()` method:
`cwRateUnit = cwMeterUnit.orElse(toStandardUnit(this.rateUnit));`

This way in the `private CloudWatchReporter(final Builder builder) {..}` you can keep the original statement `this.rateUnit = builder.cwRateUnit;`, without the need to `this.meterUnit = builder.cwMeterUnit;`. Plus, inside the `private void processMeter(..) {..}` you do not have to do in multiple places `meterUnit.orElse(rateUnit)`

Reply:
I can see what you mean.
Changed it the way you suggested it.

Reply:
Tjarko, it looks good. You may need to rebase, as I merged another PR

Reply:
One more thing I wanted to ask you - could you please update the README with the description of the new capability?

Reply:
pulled and Added description to README